infra/azure/pr-pipeline.yml

```diff
@@ -3,7 +3,7 @@ trigger:
 - master
 
 pool:
-  vmImage: 'ubuntu-20.04'
+  vmImage: 'ubuntu-22.04'
```

Comment:
Is this working for the multi item tests?

Reply:
This change should not make into the PR. Fixed.